sdk/identity/azure-identity/src/main/java/com/azure/identity/KeyringItemSchema.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+/**
+ * An expandable enum for types of item schema in a Keyring.
+ */
+
+public final class KeyringItemSchema extends ExpandableStringEnum<KeyringItemSchema> {
```

Comment:
Is this meant to be part of the public API? I'd prefer to keep these details internal.

Reply:
Good catch - It was made public to allow configurations - but since for now we are not allowing users to configure it I'll hide it.